SCClassLibrary/Common/Streams/Stream.sc

```diff
@@ -346,15 +346,21 @@ PauseStream : Stream {
 		isWaiting = true;	// make sure that accidental play/stop/play sequences
 						// don't cause memory leaks
 		era = CmdPeriod.era;
-		clock.play({
-			if(isWaiting and: { nextBeat.isNil }) {
-				clock.sched(0, this);
-				isWaiting = false;
-				this.changed(\playing)
-			};
-			nil
-		}, quant.asQuant);
-		this.changed(\userPlayed);
+		protect {
```

Comment:
I'm a bit unclear on the purpose of this `protect`? This is executing nothing in the user pattern -- everything here is under control of the player.

I think protecting inside the Routine is fine (provided that it passes tests).